submodules.json

```diff
@@ -133,5 +133,15 @@
     "name": "wordpress-boilerplate",
     "url": "https://github.com/DevinVinson/WordPress-Plugin-Boilerplate",
     "tags": ["wordpress", "website", "wordpress-plugin","web"]
+  },
+  {
+    "name": "hackathon-starter",
+    "url": "https://github.com/sahat/hackathon-starter",
```

Comment:
Hey! This repo already exists at [84](https://github.com/IEEE-VIT/templa-rs/blob/f7d3b665197d4e1bcb8c8aa8c14d56a44d2345c8/submodules.json#L84)?

Reply:
Oops I missed it, I should've double checked first!